src/content/release-notes.mdx

```diff
@@ -5,6 +5,18 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.22.1
+
+19 July 2024
+
+This versions is compatible with Kubernetes versions 1.26 to 1.29
+
+### Improvements
+
+- Updated the backend CLI to version 2.29.2 to fix critical vulnerabilities
+- Fix critical vulnerabilities in backend and installer images
+- Bump builkit image to v0.12.5-okteto3 to fix critical vulnerabilities
```

Comment:
Should we say something other than "fix" since we weren't directly affected? I don't want to freak any customers out when reading this? What do you think about changing it to:

- Updated the backend CLI to 2.29.2 to address potential vulnerabilities
- Improved the backend and installer images to enhance overall security
- Bumped Buildkit image to v0.12.5-okteto3 to fix critical vulnerabilities